apps/retrospective/services/sprint.go

```diff
@@ -306,18 +306,32 @@ func (service SprintService) getSprintTaskTypeSummary(
 }
 
 // GetSprintsList ...
-func (service SprintService) GetSprintsList(
-	retrospectiveID string,
-	userID uint) (sprints *retroSerializers.SprintsSerializer, status int, err error) {
+func (service SprintService) GetSprintsList(retrospectiveID string, userID uint, perPage string, after string) (*retroSerializers.SprintsSerializer, int, error) {
```

Comment:
format code to handle line limit. Formatter doesn't do that.
Also, why did we removed named  returns

Reply:
Go [recommends](https://github.com/golang/go/wiki/CodeReviewComments#line-length) not to break lines to keep lines short when they are more readable long.
> the wrapping would be unnecessary if they had a reasonable number of parameters and reasonably short variable names.

> In other words, break lines because of the semantics of what you're writing (as a general rule) and not because of the length of the line.

> If you find that this produces lines that are too long, then change the names or the semantics and you'll probably get a good result.